upup/pkg/fi/cloudup/bootstrapchannelbuilder.go

```diff
@@ -1088,6 +1088,22 @@ func (b *BootstrapChannelBuilder) buildAddons() *channelsapi.Addons {
 					Id:                id,
 				})
 			}
+		} else if kops.CloudProviderID(b.Cluster.Spec.CloudProvider) == kops.CloudProviderAWS {
+			key := "aws-cloud-controller.addons.k8s.io"
+			version := "v1.18.0-alpha.1"
+
+			location := key + "/v1.18.0.yaml"
+			id := "k8s-1.18-ccm"
+
+			addons.Spec.Addons = append(addons.Spec.Addons, &channelsapi.AddonSpec{
+				Name:              fi.String(key),
+				Version:           fi.String(version),
+				Manifest:          fi.String(location),
+				Selector:          map[string]string{"k8s-addon": key},
+				KubernetesVersion: ">=1.18.0",
```

Comment:
What is the correct behavior for k8s < 1.18? Previously it would install `core.addons.k8s.io`.

Reply:
Yeah, we should probably keep that behavior, as we don't have any published images pre-1.18.